src/ir/effects.h

```diff
@@ -318,8 +319,8 @@ class EffectAnalyzer {
     return hasAnything();
   }
 
-  std::set<Name> breakTargets;
-  std::set<Name> delegateTargets;
+  SmallSet<Name, 10> breakTargets;
+  SmallSet<Name, 10> delegateTargets;
```

Comment:
The changes to this file might be worth doing in a separate PR, to allow separate benchmarking.

Offhand I'm not sure if 10 is the best value here. I'd guess lower, since this makes the size of the class much larger and often we don't use any of that. But let's just benchmark various numbers here and see (I can do some of that too).

Reply:
Sure, can split it off 👍 Though I believe these were the original reason I found the allocation in std::set.
Good point on the "10".